src/main/java/seedu/address/ui/HomeCard.java

```diff
@@ -192,9 +174,9 @@ public Integer getContactAmount() {
     public double getTotalDebt() {
 
         double result = 0.0;
-        for (Person x : personList) {
-            if (x.getMoneyOwed().getAmount() > 0) {
-                result += x.getMoneyOwed().getAmount();
+        for (Person person : personList) {
+            if (person.getMoneyOwed().userOwesMoney()) {
+                result += -person.getMoneyOwed().getAmount();
```

Comment:
It's good that you changed to use the `userOwesMoney` function in `MoneyOwed`. But remember other classes should not have to know the inner workings of the `MoneyOwed` class (e.g. HomeCard shouldn't have to know negative amount means user owes money). Maybe we can have a method in `MoneyOwed` called `getAbsoluteAmount` that returns the absolute value, then HomeCard doesn't have to know/care if it's negative or not.